src/modules/service-catalog/components/useItemFormFields.tsx

```diff
@@ -108,7 +108,10 @@ export function useItemFormFields(
       if (serviceCatalogItem && serviceCatalogItem.form_id) {
         try {
           const { requestFields, associatedLookupField } =
-            await fetchTicketFields(serviceCatalogItem.form_id, baseLocale);
+            await fetchTicketFields(
+              serviceCatalogItem.form_id.toString(),
```

Comment:
Nit: I would rather change the `form_id` parameter of the function to be a number, and rename it to `formId` for coherence

Reply:
Updated. Could you re-review @Fredx87 ?